packages/rich-text-from-markdown/package.json

```diff
@@ -53,9 +53,9 @@
     "rollup-plugin-sourcemaps": "^0.6.3",
     "rollup-plugin-typescript2": "^0.36.0",
     "ts-jest": "^29.1.2",
-    "tslib": "^2.3.1",
+    "tslib": "^2.6.2",
```

Comment:
`tslib` should be a regular dep, not `devDep`. But it's not used anyway, so we should be able to remove it